### PR TITLE
Fix bugs because no one else fixes them

### LINKED ISSUE #158: Use interactions for paginator
👀

### LINKED ISSUE #168: Auto detect illegal names
The rules prohibition "unmentionable" names, which shouldn't be that hard to detect

### LINKED ISSUE #204: Profilbilder bei /rank stats
Das Profilbild des Users welcher **/rank stats** eingibt anzeigen wie bei **srep**

### LINKED ISSUE #205: Null bei /info
Bei /info kommt unter **Nutzer** null

### PR DESCRIPTION
**Pull request by DRSchlaubi**
- Optimize tag Database queries - Also query alias for auto-help tags - Increase tag column size since the message limit doesn't account for Markdown
- Make use of ephemeral commands - Make autohelp find casting tag properly - Fix tag finding
- Add BOM module gradle file
- Bump dependencies
- Fix dependency upgrade issues
- Cleanup code
- Add ability to delete command responses Co-authored-by: amejonah1200 <amejonah@gmail.com>
- Update dependencies
- Fetch Bot Guild correctly (Fix #205)
- Add user avatar to /rank stats (Fix #204)
- Use components for paginator (Fix #158)
- Add logs to make das_ happy
- Update merge-in for new changes
- Automatically sanitize unmentionable names (Fix #168)
- Improve Hikari Config
- Use coroutines for CSE

### CODE REVIEW

**Comment by DRSchlaubi** [comment]
This also supersedes #206 and #203

File: src/main/kotlin/com/github/devcordde/devcordbot/command/AbstractCommand.kt [hunk 1]
```diff
@@ -17,6 +17,8 @@
 package com.github.devcordde.devcordbot.command
 
 import com.github.devcordde.devcordbot.command.permission.Permission
+import com.github.devcordde.devcordbot.command.root.AbstractRootCommand
+import com.github.devcordde.devcordbot.command.root.AbstractSingleCommand
```
Comment (daspositioned) [inline src/main/kotlin/com/github/devcordde/devcordbot/command/AbstractCommand.kt:21]: I'm pretty sure these imports are not needed
```suggestion
```

**Review by JohnnyJayJay — DISMISSED** [review]

**Review by daspositioned — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/github/devcordde/devcordbot/command/AbstractCommand.kt (hunk 1 above)
Comment (DRSchlaubi) [reply src/main/kotlin/com/github/devcordde/devcordbot/command/AbstractCommand.kt:21]: If they weren't needed the Klimt check should have failed

Comment (daspositioned) [reply src/main/kotlin/com/github/devcordde/devcordbot/command/AbstractCommand.kt:21]: But you didn't add anything to that class besides these imports. Am I missing something here?

Comment (DRSchlaubi) [reply src/main/kotlin/com/github/devcordde/devcordbot/command/AbstractCommand.kt:21]: These are not unused, they're used in the doc comment

**Review by daspositioned — APPROVED** [review]

**Pull request merged**